src/coreclr/tools/Common/Internal/NativeFormat/NativeFormatWriter.cs

```diff
@@ -936,11 +934,9 @@ public ExternalTypeSignature(uint externalTypeId)
 
         internal override void Save(NativeWriter writer)
         {
-            NativeWriter.TypeSignatureCompressor compressor = new NativeWriter.TypeSignatureCompressor(writer);
```

Comment:
@jkotas did you see these diffs? We never implemented the compression for this but this was a stub for it. What will be left after this change is kind of nonsensical. Should we just delete or suppress?

Reply:
I have missed this one. I agree that this is nonsensical with this change.

Reply:
I deleted the stub structure and calls to related to its usage